tests/python/relay/aot/test_cpp_aot.py

```diff
@@ -138,7 +140,7 @@ def test_mobilenet(enable_usmp, target_kind):
 
     temp_dir = tvm.contrib.utils.TempDirectory()
     test_so_path = temp_dir / "test.so"
-    mod.export_library(test_so_path, cc="gcc", options=["-std=c11"])
+    mod.export_library(test_so_path, cc="c++", options=["-std=gnu++14", "-g3", "-O0"])
```

Comment:
should this also be similar to the above change?
`mod.export_library(test_so_path, cc="gcc", options=["-std=c++11", "-g3", "-O0"])`